src/services/PugQueue.js

```diff
@@ -25,35 +26,34 @@ export class PugQueue {
     this.queue.push(member)
     member.addRole(process.env.PUGS_ROLE)
 
-    return `${String.fromCodePoint(0x2705)} ${name} added to queue. ${this.getQueueState()}`
+    return `${String.fromCodePoint(0x2705)} ${name} ${roleEmoji} added to queue. ${this.getQueueState()}`
   }
 
   attemptGameStart (guild) {
     if (this.queue.length < process.env.TEAM_SIZE * 2) {
       return
     }
 
-    this.gameCounter += 1
-    const tempChannelName = `pug-${this.gameCounter}`
-
     let mentions = ``
     this.queue.forEach(member => {
-      mentions += `${member}\n`
+      const roleEmoji = getRoleEmoji(member)
+      mentions += `${member} ${roleEmoji}\n`
     })
 
-    guild.createChannel(tempChannelName, 'text').then((channel) => {
-      channel.send(`The players for this game are:\n${mentions}`)
-
-      channel.send('Nominate captains and draft teams. This channel will be deleted in an hour.')
-
-      // Start timer to delete channel
-      setTimeout(() => {
-        channel.delete()
-            .then(console.log(`Deleted channel ${tempChannelName}`))
-      }, process.env.TEMP_CHANNEL_LIFETIME)
-
-      channel.guild.channels.get(process.env.PUGS_CHANNEL).send(`Game ready to start, draft teams in channel ${channel}.`)
-    })
+    // Announce game start and alert players
+    const announceChannel = guild.client.channels.get(process.env.PUGS_ANNOUNCEMENTCHANNEL)
+    const map = getRandomMap()
+    announceChannel.send(`❮❮❮\t\t **Match Starting!**\t\t ❯❯❯\n
+*Following players head over to Match Draft:*\n
+${mentions}\n
+${map}\n
+❯❯❯\t\t*end of match announcement*\t\t❮❮❮\n
+`)
+
+    // Notification for queue full
+    const pugsChannel = guild.client.channels.get(process.env.PUGS_CHANNEL)
+    const teamSize = process.env.TEAM_SIZE * 2
```

Comment:
This is confusing, could you rather leave out this variable and put the expression where `${teamSize}` is?